doc/conf_ref.rst

```diff
@@ -201,20 +227,41 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     concurrently. Defaults to 3. The minimal number of installonly packages is 2. Value 0 or 1 means
     unlimited number of installonly packages.
 
+``installroot``
+    :ref:`string <string-label>`
+
+    The root of the filesystem for all packaging operations. It requires an absolute path. See also :ref:`--installroot commandline option <installroot-label>`.
+
+``install_weak_deps``
+    :ref:`boolean <boolean-label>`
+
+    When this option is set to True and a new package is about to be
+    installed, all packages linked by weak dependency relation (Recommends or Supplements flags) with this package will be pulled into the transaction.
+    Default is ``True``.
+
 .. _keepcache-label:
 
 ``keepcache``
     :ref:`boolean <boolean-label>`
 
     Keeps downloaded packages in the cache when set to True. Even if it is set to False and packages have not been
     installed they will still persist until next successful transaction. The default
-    is False.
+    is ``False``.
 
 ``logdir``
     :ref:`string <string-label>`
 
     Directory where the log files will be stored. Default is ``/var/log``.
 
+.. _log_rotate-label:
+
+``log_rotate``
+    :ref:`integer <integer-label>`
+
+    Log files are rotated log_rotate times before being removed. If log_rotate
+    is 0, the rotation is not performed.
+    Default is 4.
```

Comment:
Mark the `log_rotate` with backticks? Also possibly the numeric values for consistency.